test/lib/stack.spec.js

```diff
@@ -22,66 +20,76 @@ describe("stack proxy", () => {
     jest.restoreAllMocks()
   })
 
-  describe("when initialized with an cozyURL + token", () => {
-    
+  describe('when initialized with an cozyURL + token', () => {
     const params = { cozyURL, token, onCreateApp, onDeleteApp }
-    
+
     beforeAll(() => {
       jest.clearAllMocks()
-      
+
       stack.init(params)
       stack.get.app()
     })
 
-    it("should call the internal stack init", () => {
-      expect( internal.init ).toHaveBeenCalled()
+    it('should call the internal stack init', () => {
+      expect(internal.init).toHaveBeenCalled()
     })
 
-    it("should not call the cozy-client stack init", () => {
-      expect( client.init ).not.toHaveBeenCalled()
+    it('should not call the cozy-client stack init', () => {
+      expect(client.init).not.toHaveBeenCalled()
     })
 
-    it("should forward requests to the internal stack client", () => {
-      expect( internal.get.app ).toHaveBeenCalled()
+    it('should forward requests to the internal stack client', () => {
+      expect(internal.get.app).toHaveBeenCalled()
     })
 
-    it("should not forward requests to the cozy-client stack client", () => {
-      expect( client.get.app ).not.toHaveBeenCalled()
+    it('should not forward requests to the cozy-client stack client', () => {
+      expect(client.get.app).not.toHaveBeenCalled()
     })
-
   })
 
-  describe("when initialized with a cozy-client instance", () => {
-
-    const cozyClient = { 
+  describe('when initialized with a cozy-client instance', () => {
+    const cozyClient = {
       getStackClient: () => {
         return {
-          token: { token: "mytoken"},
-          uri: "https://test.mycozy.cloud"
+          token: { token: 'mytoken' },
+          uri: 'https://test.mycozy.cloud'
         }
       }
     }
     const params = {
       cozyClient,
       onCreateApp: function() {},
-      onDeleteApp: function() {},
+      onDeleteApp: function() {}
     }
 
     beforeAll(() => {
       jest.clearAllMocks()
-      
+
       stack.init(params)
       stack.get.app()
     })
 
-    it("should call the client stack init", () => {
-      expect( client.init ).toHaveBeenCalled()
+    it('should call the client stack init', () => {
+      expect(client.init).toHaveBeenCalled()
     })
 
-    it("should forward requests to the cozy-client stack client", () => {
-      expect( client.get.app ).toHaveBeenCalled()
+    it('should forward requests to the cozy-client stack client', () => {
+      expect(client.get.app).toHaveBeenCalled()
     })
-    
```

Comment:
It seems this commit also contains linting, it would be nice to separate the two commits.

Reply:
👍 on this for the next time.